### ISSUE TITLE
Some LLVM versions are not available for Ubuntu

### ISSUE THREAD

**Issue by ADKaster** [aminya/setup-cpp#144]
I added the following to my action that is using ubuntu 22.04 (jammy):
```
    - uses: aminya/setup-cpp@v0.22.0
      with:
        llvm: 14.0.6
```

And instead of installing llvm 14.0.6, or even the claimed fallback of llvm 15.0.2, it tries to install gcc 11, and fails.

```
Installing llvm 14.0.6
  Installing llvm 14.0.6 x64 via direct downloading
  Warning: Falling back to LLVM version 15.0.2 -rhel86 for the Ubuntu.
  Installing gcc 11 via apt-get
```

The URL that the action tries to pull from the llvm releases page has not been published since llvm 14.0.0:

https://github.com/llvm/llvm-project/releases/tag/llvmorg-14.0.0 has 62 assets, while  https://github.com/llvm/llvm-project/releases/tag/llvmorg-14.0.6 only has 44. 

That is to say, https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.6/clang+llvm-14.0.6-x86_64-unknown-linux-gnu-rhel86.tar.xz does not exist.

So there are two issues:

- Pulling x86_64 ubuntu packages from llvm-project github releases does not work anymore
- The fallback to 15.0.2 failed, and installed gcc 11 instead, which is not an equivalent compiler. (it should probably be gcc 12)

**Comment by aminya** [comment]
On Ubuntu only 14.0.0 is available.
https://github.com/aminya/setup-cpp/blob/08aa9c845a12786d832df28189a9a2adbb6cc5a7/src/llvm/llvm_url.ts#L129

**Comment by aminya** [comment]
Another source of the binaries for LLVM is the apt repositories. We can use that as a fallback for Ubuntu.
https://apt.llvm.org/

**Comment by aminya** [comment]
Fixed by #182
